api-cursors/src/main/java/org/zalando/nakadi/service/CursorsService.java

```diff
@@ -254,33 +245,4 @@ private void validateCursorsBelongToSubscription(final Subscription subscription
             throw new UnableProcessException("Event type does not belong to subscription: " + wrongEventTypes);
         }
     }
-
-    private class SubscriptionCursorComparator implements Comparator<SubscriptionCursorWithoutToken> {
```

Comment:
is it just clean up due to the fact that zero version cursor is not used anymore ?

Reply:
No, because this class is wrong and not needed